main.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <iomanip>
+
+int main() {
+
+    std::cout << "JOE'S ACME SOFTWARE, INC. STOCK VALUES\n\n"; // just some header text
+
+    //stock purchase calculations
+    const int SHARES_BOUGHT = 1000;
+    const double PURCHASE_PRICE = 45.50;
+    double total_stock_price = SHARES_BOUGHT * PURCHASE_PRICE; // price of all stock
```

Comment:
This end of line comment is not necessary. You did a great job of variable naming making the comment redundant.